### PR TITLE
[ramda] Bump `types-ramda` package to latest

### PR DESCRIPTION
**Pull request by Harris-Miller**
This MR
* updates tests that use `propEq`, as ramda@0.29.0 swapped the argument order
* Adds new re-exports for new `modifyPath` function
* Bumps `types-ramda` to latest

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## ramda ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ramda@latest/))
was missing the following properties:
1. addIndexRight
1. sequence
1. swap
1. unwind
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 6fff80052682a476182795c11c259b7ad3aaa902
</p>

**Comment by Harris-Miller** [comment]
@TheHandsomeCoder @donnut @mdekrey @sbking @afharo @teves-castro @hojberg @samsonkeung @angeloocana @raynerd @moshensky @ethanresnick @deftomat @blimusiek @biern @rayhaneh @rgm @drewwyatt @jottenlips @minitesh @Krantisinh @Nemo108 @jituanlin @Philippe-mills @Saul-Mirone @Nicholaiii @LORDBABUINO @couzic @NEWESTERS @adispring @essenmitsosse 

Please let me know if you would like your @'s removed

**Comment by afharo** [comment]
@Harris-Miller given that the reviews from now on are only based on bumping the package version, I'd appreciate if I'm removed from the approvers list 🤗

File: types/ramda/index.d.ts [hunk 1]
```diff
@@ -4,7 +4,6 @@
 //                 Erwin Poeze <https://github.com/donnut>
 //                 Matt DeKrey <https://github.com/mdekrey>
 //                 Stephen King <https://github.com/sbking>
-//                 Alejandro Fernandez Haro <https://github.com/afharo>
```
Comment (afharo) [inline types/ramda/index.d.ts:7]: 🧡

**Comment by Harris-Miller** [comment]
@TheHandsomeCoder @donnut @mdekrey @sbking @afharo @teves-castro @hojberg @samsonkeung @angeloocana @raynerd @moshensky @ethanresnick @deftomat @blimusiek @biern @rayhaneh @rgm @drewwyatt @jottenlips @minitesh @Krantisinh @Nemo108 @jituanlin @Philippe-mills @Saul-Mirone @Nicholaiii @LORDBABUINO @couzic @NEWESTERS @adispring @essenmitsosse

Sorry for the spam, I decided to do this in the other direction. I'm removing everyone except a select few. Since we moved all the type definitions to https://github.com/ramda/types, it doesn't really matter anyhow.

@Nicholaiii keeping you since you've been the sole approval for the past year, also adding @kedashoe @adispring who are part of the Ramda org.

Let me know if you want to be added to https://github.com/ramda/types as an approval and I'll be happy to add you over there

**Comment by Nicholaiii** [comment]
Do feel free to add me to other repo.

**Comment by Harris-Miller** [comment]
@Nicholaiii I added you with write access to [types/ramda](https://github.com/ramda/types). And if you could approve this PR for me, please and thank you!

**Review by Nicholaiii — APPROVED** [review]

**Review by afharo — APPROVED** [review]

**Review by gabritto — APPROVED** [review]

**Comment by Harris-Miller** [comment]
Ready to merge

**Pull request merged**